input/input.c

```diff
@@ -275,11 +275,20 @@ static struct mp_cmd *queue_peek_tail(struct cmd_queue *queue)
     return cur;
 }
 
+static int queue_cmd(struct input_ctx *ictx, mp_cmd_t *cmd)
+{
+    if (cmd) {
+        queue_add_tail(&ictx->cmd_queue, cmd);
+        mp_input_wakeup(ictx);
+    }
+    return 1;
+}
+
 static void append_bind_info(struct input_ctx *ictx, char **pmsg,
                              struct cmd_bind *bind)
 {
     char *msg = *pmsg;
-    struct mp_cmd *cmd = mp_input_parse_cmd(ictx, bstr0(bind->cmd),
+    struct mp_cmd *cmd = mp_input_parse_cmd_str(ictx->log, bstr0(bind->cmd),
```

Comment:
Why this unrelated change mixed in this commit?

Reply:
They look the same too me. `mp_input_parse_cmd` is not public in fact, the `mpv_command_string` is public, no?

Also `mp_input_parse_cmd` and `mp_input_parse_cmd_str` both doesn't lock, so the only difference is the first argument.